VSRAD.Syntax/Parser/Parser.cs

```diff
@@ -50,5 +70,24 @@ public static IBlock SetBlockReady(IBlock block, List<IBlock> list)
         }
 
         public abstract List<IBlock> Parse(IEnumerable<TrackingToken> trackingTokens, ITextSnapshot version, CancellationToken cancellation);
+
+        protected void AddExternalDefinitions(List<KeyValuePair<AnalysisToken, ITextSnapshot>> definitions, TrackingToken includeStr, ITextSnapshot version)
+        {
+            try
+            {
+                var filePath = Path.Combine(_documentInfo.DirectoryPath, includeStr.GetText(version).Trim('"'));
+                var documentAnalysis = _documentAnalysisProvoder.GetOrCreateDocumentAnalysis(filePath);
+                if (documentAnalysis != null)
+                {
+                    foreach (var funcToken in documentAnalysis.LastParserResult.GetGlobalTokens())
+                    {
+                        definitions.Add(new KeyValuePair<AnalysisToken, ITextSnapshot>(funcToken, documentAnalysis.CurrentSnapshot));
+                    }
+                }
+            }
+            catch (ArgumentException)
+            {
```

Comment:
Ditto, the catch should either do something or be nonexistent, possibly replaced by a check.